lib/graphql/transaction.ts

```diff
@@ -5,50 +5,58 @@ import {
   TransactionsConnectionEdge
 } from "./schema";
 import { Model } from "./model";
+import { FetchOptions } from "./types";
+import { ResultPage } from "./resultPage";
+
+const safeGet = (source: any, path: any, defaultValue: any) =>
+  get(source, path, defaultValue) || defaultValue;
 
 export class Transaction extends Model {
-  async fetchAll(): Promise<Array<TransactionEntry>> {
-    const query = `{
-      viewer {
+  async fetch(args?: FetchOptions): Promise<ResultPage> {
+    const query = `
+      query fetchTransactions ($first: Int, $last: Int, $after: String, $before: String) {
+        viewer {
           mainAccount {
-            transactions {
+            transactions(first: $first, last:$last, after: $after, before: $before) {
               edges {
                 node {
-                  id
-                  amount
-                  name
-                  iban
-                  type
-                  bookingDate
-                  valutaDate
-                  originalAmount
-                  foreignCurrency
-                  e2eId
-                  mandateNumber
-                  paymentMethod
-                  category
-                  userSelectedBookingDate
-                  purpose
-                  bookingType
-                  invoiceNumber
-                  invoicePreviewUrl
-                  invoiceDownloadUrl
-                  documentType
```

Comment:
Did we loose them? Should be `documentNumber`, `documentPreviewUrl`, `documentDownloadUrl`, ...

Reply:
true, it was renamed